src/tools/ceph_dedup_tool.cc

```diff
@@ -179,6 +181,12 @@ po::options_description make_usage() {
     ("snap", ": deduplciate snapshotted object")
     ("debug", ": enable debug")
     ("pgid", ": set pgid")
+    ("object-dedup-threshold", po::value<uint32_t>(), ": set the threshold for object dedup (percentile)")
+    ("chunk-dedup-threshold", po::value<uint32_t>(), ": set the threshold for chunk dedup")
+    ("sampling-ratio", po::value<int>(), ": set the sampling ratio (percentile)")
+    ("deamon", ": execute sample dedup in deamon mode")
+    ("iterative", ": execute sample dedup iteratively")
```

Comment:
Rename this to "loop".  Rename related variables and params as well.

("loop", ": execute sample-dedup in a loop until terminated.  Sleeps 'wakup-period' seconds between iterations.")

Reply:
Done. Also, check_whole_object_dedupable() (including invoking tier-flush) is removed, which means that only deduplicating chunks remains.